dev/tasks/exec/utils/filtergitignore.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const fs = require( 'fs' );
+const filter = require( 'gulp-filter' );
+const parseGitignore = require( 'parse-gitignore' );
+const PassThrough = require( 'stream' ).PassThrough;
+
+module.exports = function filterGitignore() {
+	const fp = '.gitignore';
+
+	if ( !fs.existsSync( fp ) ) {
+		return new PassThrough( { objectMode: true } );
+	}
+
+	let glob = parseGitignore( fp );
```

Comment:
`const` in both cases. Don't use `let` unless you expect the variable to change.